### ISSUE TITLE
IE11 syntax error due to destructuring

### ISSUE THREAD

**Issue by afisher88** [sweetalert2/sweetalert2#1391]
## Current behavior
Using version 8.0.1 in IE11 I get a syntax error in the SweetAlert constructor due to destructuring. This causes my app not to load.

![swal-error2](https://user-images.githubusercontent.com/5710575/51848796-2b0c7480-2316-11e9-9119-965e05d4929f.JPG)

![swal-error1](https://user-images.githubusercontent.com/5710575/51848800-2cd63800-2316-11e9-9a54-cc5074c0e738.JPG)

## Expected behavior
Shouldn't error in IE11

## Live demo
I can't replicate this issue with your template as it does seem to work in IE11. But using the latest version in my sandbox repo I get the same issue:

https://github.com/afisher88/sweetalert2-sandbox

I can get around this by transpiling my node_modules folder but I don't really want to have to do this.

```
    rules: [
      {
        test: /\.(js|jsx)$/,
        exclude: /(node_modules)/, // Remove this line
        use: {
          loader: 'babel-loader',
          options: {
            presets: ['@babel/preset-env', '@babel/preset-react'],
            plugins: [
              '@babel/plugin-proposal-object-rest-spread',
              '@babel/transform-spread',
              '@babel/plugin-proposal-class-properties'
            ]
          }
        }
      },
```

**Comment by gverni** [comment]
Hi @afisher88 your problem should be the same discussed in #1390. Currently the workaround is to add the following in webpack.config.js: 

```
 resolve: {
    mainFields: [
      'main',
      'module'
    ]
  },
```

I tested on your sandbox and it works. Don't forget to add the polyfill for promises

**Comment by limonte** [comment]
I am going to remove the module field from `package.json`, so the webpack workaround suggested above won't be needed: #1392

**Comment by limonte** [comment]
Thank you for reporting @afisher88 
Thank you for investigating @gverni 

The issue should be fixed in the latest release (8.0.2), please check it out. 

Apologies about this inconvenience!

**Comment by afisher88** [comment]
No problem, thanks for fixing so quickly!
